src/talk/renderer/Viewer/ViewerHandlerBase.vue

```diff
@@ -0,0 +1,91 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+  -->
+
+<script setup>
+import { computed } from 'vue'
+import { t } from '@nextcloud/l10n'
+import NcEmptyContent from '@nextcloud/vue/dist/Components/NcEmptyContent.js'
+import NcLoadingIcon from '@nextcloud/vue/dist/Components/NcLoadingIcon.js'
+import IconAlertCircleOutline from 'vue-material-design-icons/AlertCircleOutline.vue'
+
+const props = defineProps({
+	loading: {
+		type: Boolean,
+		required: false,
+		default: false,
+	},
+	error: {
+		type: [String, Boolean],
+		required: false,
+		default: false,
+	},
+	/**
+	 * In lazy mode the content is rendered visually hidden during loading.
+	 * Used when loading happens on rendering, for example, loading <img>.
+	 */
+	lazy: {
+		type: Boolean,
+		required: false,
+	},
+})
+
+const readyToShow = computed(() => !props.loading && !props.error)
+</script>
+
+<template>
+	<div class="viewer-wrapper">
+		<NcEmptyContent v-if="loading" :name="t('talk_desktop', 'Loading …')" class="delayed-appear">
+			<template #icon>
+				<NcLoadingIcon />
+			</template>
+		</NcEmptyContent>
+
+		<NcEmptyContent v-else-if="error"
+			:name="t('talk_desktop', 'Could not load the file')"
+			:description="typeof error === 'string' ? error : undefined">
+			<template #icon>
+				<IconAlertCircleOutline />
+			</template>
+		</NcEmptyContent>
```

Comment:
Almost not visible

![image](https://github.com/user-attachments/assets/a9842e13-cb61-46d3-8096-a384b7684dd9)

Reply:
Fixed in the following commit, thanks!